config.toml

```diff
@@ -1,5 +1,5 @@
 [logging]
-debug = false
+debug = true
```

Comment:
I'm afraid this will eventually get to stage/prod increasing the logs -> AWS costs

Reply:
removing =)